Documentation/hacking.md

```diff
@@ -207,7 +207,47 @@ Now, GOTO [saving](#saving-the-set-of-dependencies)
 
 #### Removing an existing dependency
 
-This is the simplest case of all: simply remove all references to a dependency from the source files.
+This is almost the simplest case of all: simply remove all references to a dependency from the source files.
+
+Now, GOTO [saving](#saving-the-set-of-dependencies)
+
+#### Add a new application
+
+In this case you'll first need to retrieve the application you're working with into `GOPATH`.
+As a simple example, assuming we're adding `github.com/fizz/buzz/app`:
+
+```
+$ go get -d github.com/fizz/buzz
+```
+
+Then add you new application to the vendoredApps file:
+
+```
+$ vim vendoredApps
+...
+github.com/fizz/buzz/app
+...
+```
+
+Now, GOTO [saving](#saving-the-set-of-dependencies)
+
+#### Update an existing application
```

Comment:
Hm, this is all identical to the normal dependency updating, does it need to be a separate section?